### PR TITLE
midnightcontrols is now on Modrinth!

### PR DESCRIPTION
**Pull request by DioEgizio**

### CODE REVIEW

File: src/wiki/getting-started/controller-support.md [hunk 1]
```diff
@@ -38,17 +38,17 @@ Remember to get the mapping using [**the app linked in the mod**](https://genera
...
+In fact, here we're recommending [MidnightControls](https://modrinth.com/mod/midnightcontrols).
 
 It's an updated fork of LambdaControls.
 
-To install this mod on PolyMC, download it directly [from their GitHub releases](https://github.com/TeamMidnightDust/MidnightControls/releases) and drag-and-drop the jar in PolyMC. 
+It can be be installed using PolyMC's mod downloader funtion With Modrinth.
 
 Once installed, please launch your instance and navigate to the in-game controls menu.
 
 Within the in-game controls menu, you may need to change the "Mode" setting to **Controller**, in order for the game to respond to input from the gamepad.
```
Comment (Scrumplex) [inline src/wiki/getting-started/controller-support.md:49]: yes

**Review by Scrumplex — CHANGES_REQUESTED** [review]

File: src/wiki/getting-started/controller-support.md (hunk 1 above)
Comment (DioEgizio) [reply src/wiki/getting-started/controller-support.md:49]: ?

Comment (Scrumplex) [reply src/wiki/getting-started/controller-support.md:49]: yes

**Review by Scrumplex — CHANGES_REQUESTED** [review]
yes

**Review by Scrumplex — CHANGES_REQUESTED** [review]
here i re-reviewed

**Comment by DioEgizio** [comment]
stop it lmao, scrumplex moment

**Review by Scrumplex — CHANGES_REQUESTED** [review]
this is completely wrong

**Pull request merged**